Src/Base/AMReX_Vector.H

```diff
@@ -16,6 +16,7 @@
 #include <thrust/iterator/zip_iterator.h>
 #include <thrust/device_ptr.h>
 #include <thrust/tuple.h>
+#include <curand_kernel.h>
```

Comment:
Does this need to be here? I don't see anything in AMReX_Vector.H that uses this. Maybe it should go in AMReX_Utility.H instead?